docs/en/protocols/HTTP-API-Protocol.md

```diff
@@ -0,0 +1,198 @@
+# HTTP API Protocol
+
+HTTP API Protocol define the API data format, including api request and response data format.
+
+### Do register
+
+For more information about data format can be found in [Register service](https://github.com/apache/skywalking-data-collect-protocol/tree/master/register/Register.proto).
```

Comment:
Seems to be a grammar error

For more information about the data format, please refer to .....

OR

Detail information about data format can be found in ......